.gitignore

```diff
@@ -3,7 +3,6 @@ node_modules
 /build
 /.svelte-kit
 /package
-.env
```

Comment:
This file should be named example.env and referenced with a flag (typically `-r ./example.env), otherwise a person can't ignore their real .env from a copy of the repo.

Reply:
Just renamed the current .env file to .env.local, had to include .env.local to prevent any conflicts with the npm run dev and npm run build commands. 

Let me know if it still causes an issue on your end. Additionally, if you have any suggestions on how we can handle this more effectively, I'd be happy to incorporate them.

Reply:
On second thought, It's probably safer to remove all .env files to prevent future conflicts. Renamed .env.local to .env.example.